### PR TITLE
smooth transitions added

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,57 +2,117 @@
...
+              class="nav__link"
+            >
+              Apple
+            </a>
           </li>
 
           <li class="nav__item">
-            <a href="#sampsung" class="nav__link">Samsung</a>
+            <a
+              href="#sampsung"
```
Comment (UmizDemud) [inline src/index.html:57]: _samsunq_

**Review by UmizDemud — APPROVED** [review]
There is a typo as I noted.

Other thant that,
Good job!

File: src/index.html (hunk 1 above)
Comment (lefteris-L) [reply src/index.html:57]: My bad. Thanks for the reviews

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
-@import "blocks/page";
-@import "blocks/header";
-@import "blocks/catalog";
+@use "./blocks/page.scss";
+@use "./blocks/header.scss";
+@use "./blocks/catalog.scss";
```
Comment (meforoma) [inline src/styles/main.scss:3]: did `@import` served you bad?

**Review by meforoma — APPROVED** [review]
nice-o 👍

**Comment by lefteris-L** [comment]
> nice-o 👍

Thanks! @meforoma

**Pull request closed without merging**